cli/popper/commands/cmd_init.py

```diff
@@ -10,7 +10,12 @@
 def cli(ctx):
     """Initializes a repository by creating the .popper.yml file.
     """
-    project_root = scm.get_root_folder()
+    try:
+        repo = git.Repo(search_parent_directories=True)
+    except git.exc.InvalidGitRepositoryError:
+        pu.fail('Must be a git repository ! \n')
+
+    project_root = repo.git.rev_parse('--show-toplevel')
```

Comment:
can we move this functionality to `scm`? In this way this can be shared with other commands that might want to do this (like `scaffold`).

Reply:
@ivotron Ok